.pre-commit-config.yaml

```diff
@@ -0,0 +1,31 @@
+fail_fast: true
+
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: debug-statements
+
+-   repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+    -   id: black
+        args: [--line-length=79]
```

Comment:
Question (here, and for other repos below): do these tools not just take settings from `pyproject.toml` already? When I run `black`, I type `poetry run black .`, and then black takes settings from the pyproject file. Is that different when running from precommit?

Reply:
Yes this works! I thought the pre-commit did not access it, but I just verified that putting the black/isort/mypy configs in `pyproject.toml` still works.